Ports/.port_include.sh

```diff
@@ -412,6 +412,11 @@ func_defined pre_configure || pre_configure() {
     :
 }
 func_defined configure || configure() {
+    if [[ "$configscript" == "cmake" ]]; then
```

Comment:
if we go this route, it's important to make sure that the -S argument is the `basename` of that path. (relative to port src dr etc etc)

Reply:
> port src dr 
@ADKaster you mean `${PORT_BUILD_DIR}/${workdir}` or just `${PORT_BUILD_DIR}`

Reply:
Now I got it

Reply:
Yes, @timschumi  your point makes sense. Since it has `script` in name, it should be a human readable file, not binary file like `cmake`. I will do the changes.

Reply:
This is done.